src/components/ContactUsForm/index.jsx

```diff
@@ -22,41 +23,44 @@ const ContactUsForm = () => {
   // The change in isSubmitted state triggers post submission message
   const [isSubmitted, setIsSubmitted] = useState(false);
 
+  // The change in isLoading state triggers SyncOutlined icon spinning
+  const [isLoading, setIsLoading] = useState(false);
+
   const handleChange = (e) => setFormValue({ [e.target.name]: e.target.value });
 
   // application/x-www-form-urlencoded stands for POST data at HTTP.
   // body represents HTTP message and send data to server in a query string format.
   // encode is used for passing the string to the body
   // setTimeOut stands for showing post submission message about a time after submit
   const handleSubmit = (e) => {
+    setIsLoading(true);
+    e.preventDefault();
     fetch("/", {
       method: "POST",
       "data-netlify": "true",
       headers: { "Content-Type": "application/x-www-form-urlencoded" },
       body: encodeFormData({ "form-name": "contact", ...formValue }),
-    }).then(() => {
-      setTimeout(() => setIsSubmitted(false), 2000);
-      setIsSubmitted(true);
-      setFormValue({
-        name: "",
-        email: "",
-        message: "",
-      }).catch((error) => alert(error));
-
-      e.preventDefault();
-    });
+    })
+      .then(() => {
+        setIsLoading(false);
+        setIsSubmitted(true);
+        message.success({
+          content: t("contact.contactUsForm.postSubmissionMessage"),
+          className: "contactUsForm__postSubmissionMessage",
+        });
+        setFormValue({
+          name: "",
+          email: "",
+          message: "",
+        });
+      })
+      .catch((error) => alert(error));
```

Comment:
message.error(error)

Reply:
Thanks for your review, I updated error function, PTAL.